.github/workflows/stale.yml

```diff
@@ -0,0 +1,49 @@
+name: Close inactive issues and PRs
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
```

Comment:
Any specific reason why you didn't use the latest version (v6.0.1)?

Reply:
No reason. I adopted the code from <https://docs.github.com/en/actions/managing-issues-and-pull-requests/closing-inactive-issues>.